internal/schema2/service_properties.go

```diff
@@ -0,0 +1,16 @@
+/*
+ * HCS API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: 2.4
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package hcsschema
+
+import "encoding/json"
+
+type ServiceProperties struct {
+	Properties []json.RawMessage `json:"Properties,omitempty"` // Was originally []interface{}
```

Comment:
I would say if you're going to have a comment here it should be about why you needed to change from []interface{} to []json.RawMessage, unless it was just part of the schema's own update, in which case, I'm not sure a comment is needed.

Reply:
Done